codegen/CheckerUtils/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>hkust.org</groupId>
+    <artifactId>CheckerUtils</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I think the version of `maven-compile-plugins` should be added in all `pom.xml`. Otherwise, the idea will always output warning information.

Reply:
Also, I cannot run the `maven install` successfully in the IDEA after open the project.

Reply:
Can you have a look at our discussion?  Also, after importing the project from an external source, the modules in IDEA might be wrong, in that case, you need to manually remove the main/test module and set the two folders as the main/test folder of module codegen.